config/config-template-long.yaml

```diff
@@ -174,4 +174,27 @@ scanners:
         auto_create_context: False  # Optional. set to True to auto-create engagement (requires product_name and engagement_name)
 
 
-# Other scanners to be defined(TBD)
+  generic:
+    # This is a generic scanner, defined by the user
+
+    # file or directory where results are stored on the host.
+    results: "/path/to/results"   # if None or "*stdout", the command's standard output is selected
+
+    container:
+      type: "podman"    # currently, only podman is supported
+      parameters:
+        # Mandatory: image to run.
+        image: "docker.io/aquasec/trivy" # Example: for a trivy scan
+
+        # Optional: inject in an existing Pod
+        podName: "mypod"
+
+        # Optional: command to run, as a list of arguments. By default, the image's entrypoint will be run
+        command: ["image", "<image to scan>"]   # example for a trivy scan.
```

Comment:
updated. 
Also used shlex.split() so that we can use either a string (which will be split), or a list: being forced to use a list made the config a little annoying to write and read.